packages/notifi-react-card/lib/components/NotifiEmailInput.tsx

```diff
@@ -48,9 +48,11 @@ export const NotifiEmailInput: React.FC<NotifiEmailInputProps> = ({
 
   const { email: emailErrorMessage } = formErrorMessages;
 
-  const { resendEmailVerificationLink } = useNotifiSubscribe({
-    targetGroupName: 'Intercom',
-  });
+  const subscribe = demoPreview
+    ? null
+    : useNotifiSubscribe({
+        targetGroupName: 'Intercom',
+      });
```

Comment:
hooks cannot be rendered conditionally, otherwise React gets angry -- can we make `useNotifiSubscribe` depend on the Subscription Context and handle the network issues in there?